class/defaults.yml

```diff
@@ -1,3 +1,11 @@
 parameters:
   metallb:
-    namespace: syn-metallb
+    namespace: metallb-system
+    secretkey: none 
+    # ?{vaultkv:${customer:name}/${cluster:name}/metallb/secretkey}
+    # kubectl create secret generic -n metallb-system memberlist --from-literal=secretkey="$(openssl rand -base64 128)"
+    protocol: layer2
+    addresses: 
+      - x.x.x.x/32
```

Comment:
Defaults must be functional values.

If no default value can be given, Omit them and let the compile fail. Mention in documentation.

Reply:
Current default is `null`. It's documented to provide this value.

Reply:
Fixed as well as in the feature branch #2